69.md

```diff
@@ -0,0 +1,126 @@
+
+## Purpose
+
+The purpose of poll notes is to conduct quantitative public opinion polls over nostr by requiring voters pay to participate. By tying results to real satoshi valuations, nostr polls intend to provide superior signal compared to other free polling models. Imposing real monetary costs for participation should also discourage undesired attempts to sway results.
+
+Pollers may specify multiple `p` keys, to allow participants to choose which recipient they zap, regardless of their vote choice. The option for such distribution of funds should also incentivize participation by increasing voter optionality, poller authenticity, and legitimacy of results by mitigating certain unauthentic attack vectors.
+
+By including a `value_maximum` limit, polls can stop single 'whale' votes discounting many smaller 'shrimp' votes. Likewise, by including a `value_minimum` limit, polls can make automated low-value vote flooding attacks prohibitively expensive. However, both limits remain optional to allow for freedom in poll design.
+
+By setting `value_maximum` and `value_minimum` equal, a more traditional style poll may be designed, which weights each vote equal and limits each participant to a single vote.
+
+The optional `consensus_threshold` is intended as a simple 'measuring bar', defined as the minimum percentage for any single `poll_option`'s percentage of the total tally to attain poll consensus status.
+
+A careful balancing of all poll attributes should enable pollers to conduct tailored polls that deliver meaningful and valuable outcomes.
+
+## Zap poll format
+
+A poll event:
+* MUST specify 1 or more `p` tags, each including the same primary hosting relay
+* if an `e` tag is specified, it:
+  * MUST include the same primary hosting relay URL as the `p` tags
+* MUST contain a primary description string, specified in the `content` field
+* MUST contain at least 2 `poll_option` tags, each specifying an index and a unique description string, formatted as below
+* SHOULD specify a `closed_at` time:
+  * when specified, MUST include a valid [`ots` tag](03.md), proving original poll publishing time
+  * a `closed_at` value of null or less than or equal to the `created_at` field indicates a poll SHOULD NOT be closed.
+* MAY specify a `value_maximum` satoshi value for zapped votes to be included in the tally
+* MAY specify a `value_minimum` satoshi value for zapped votes to be included in the tally
+  * `value_minimum` MUST be less than or equal to `value_maximum`, when both are specified
+* MAY include a `consensus_threshold` (0-100), representing a percentage threshold for any single vote option to achieve poll consensus
```

Comment:
Perhaps clarify that `consensus_threshold` applies only if `closed_at` was specified and only after that time has passed. Otherwise, would not the first vote cross any specified threshold immediately? Or can the threshold be crossed and then uncrossed? Not sure what this means.

Reply:
When no `closed_at` specified, `consensus_threshold` will fluctuate with every new vote